slack_bolt/context/ack/internals.py

```diff
@@ -48,6 +78,14 @@ def _set_response(
             body["options"] = convert_to_dict_list(body["options"])
         if "option_groups" in body:
             body["option_groups"] = convert_to_dict_list(body["option_groups"])
+        if "response_type" in body:
```

Comment:
I'm not sure what 82 and 84 are doing. Don't these keys already exist as the assigned value already?

Reply:
Good catch! I will remove these lines.